content/docs/concepts/security/index.md

```diff
@@ -371,23 +375,24 @@ spec:
   - services: ["products.default.svc.cluster.local"]
     methods: ["GET", "HEAD"]
     constraints:
-    - key: "version"
+    - key: request.headers[“version”]
       values: ["v1", "v2"]
 {{< /text >}}
 
-#### `ServiceRoleBinding`
+#### ServiceRoleBinding
 
-A `ServiceRoleBinding` specification includes two parts:
-* **roleRef** refers to a `ServiceRole` resource **in the same namespace**.
-* A list of **subjects** that are assigned the role.
+A ServiceRoleBinding specification includes two parts:
+* **roleRef** refers to a ServiceRole resource in the same namespace.
+* A list of **subjects** that are assigned to the role.
 
-A subject can either be a "user", or a "group", or is represented with a set of "properties". Each entry ("user" or "group" or an entry
-in "properties") must match one of fields ("user" or "groups" or an entry in "properties") in the "subject" part of the "requestcontext"
-instance.
+A subject can be either an explicitly specified “user”, or represented by a set of “properties”.  A “property” in a ServiceRoleBinding
+“subject” is similar to “constraints” in a ServiceRole, in that it lets you use conditions to specify a set of accounts that should
+be assigned to this role. It contains “key” and allowed “values”, where supported “key” are listed in the
+[“constraints and properties”](https://preliminary.istio.io/docs/reference/config/authorization/constraints-and-properties/) page.
```

Comment:
should use `/docs/reference/config/authorization/constraints-and-properties/` instead of `https://preliminary.istio.io/docs/reference/config/authorization/constraints-and-properties/`